integration_tests/src/main/python/json_fuzzing_test.py

```diff
@@ -0,0 +1,415 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A JSON generator built based on the context free grammar from https://www.json.org/json-en.html
+
+import random
+from typing import List
+from data_gen import *
+from asserts import assert_gpu_and_cpu_are_equal_collect
+from marks import approximate_float
+
+_name_gen = StringGen(pattern= "[a-z]{1,30}",nullable= False)
+_name_gen.start(random)
+_string_gen = StringGen(pattern= "[a-z]{1,30}",nullable= False)
```

Comment:
We should include a wider range of characters in the generated strings. Spark supports parsing dates and times from JSON strings so we should include characters such as digits and symbols such as ':', ".", and '-'. We may want to also have specific fuzzer code for generating dates and timestamps as strings.

Reply:
We should also include whitespace characters such as `\n`, `\r`, `\t`, and ' '. Perhaps we can just use the full ASCII character set.

Reply:
@andygrove  The current generator just support generating integer, fp, and string. In the future, I will add more data types such as date, timestamp and map.

Reply:
> We should also include whitespace characters such as `\n`, `\r`, `\t`, and ' '. Perhaps we can just use the full ASCII character set.

Done